### PR TITLE
[OSDOCS-6377] Update kuryr-ovn kubernetes migration docs

### PR DESCRIPTION
**Pull request by gryf**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-6377

Link to docs preview: https://62843--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-kuryr-cleanup.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ plugin, you must clean up the resources that Kuryr created previously.
 The clean up process relies on a Python virtual environment to ensure that the package versions that you use support tags for Octavia objects. You do not need a virtual environment if you are certain that your environment uses at minimum:
 * `openstacksdk` version 0.54.0
 * `python-openstackclient` version 5.5.0
-* `python-octaviaclient` version 2.3.0 
+* `python-octaviaclient` version 2.3.0
```
Comment (maxwelldb) [inline modules/nw-kuryr-cleanup.adoc:17]: Can't directly make suggestions on deleted lines for whatever reason, but I think this needs to be restructured to make the list work.

```
The clean up process relies on a Python virtual environment to ensure that the package versions that you use support tags for Octavia objects. You do not need a virtual environment if you are certain that your environment uses at minimum:

* `openstacksdk` version 0.54.0

* `python-openstackclient` version 5.5.0

* `python-octaviaclient` version 2.3.0
```

Comment (gryf) [reply modules/nw-kuryr-cleanup.adoc:17]: oh, right.

File: modules/nw-kuryr-cleanup.adoc [hunk 2]
```diff
@@ -46,13 +49,13 @@ $ source /tmp/venv/bin/activate
...
 ----
-(venv) $ pip install pip --upgrade
+(venv) $ pip install --upgrade pip
 ----
 .. Install the required Python packages by running the following command:
 +
 [source,terminal]
 ----
-(venv) $ pip install openstacksdk==0.54.0 python-openstackclient==5.5.0 python-octaviaclient==2.3.0
+(venv) $ pip install openstacksdk==0.54.0 python-openstackclient==5.5.0 python-octaviaclient==2.3.0 'python-neutronclient<9.0.0'
```
Comment (gryf) [inline modules/nw-kuryr-cleanup.adoc:58]: BTW, should a note be added why we are preventing installing python-neutronclient in version 9 or higher?

Comment (maxwelldb) [reply modules/nw-kuryr-cleanup.adoc:58]: If it's helpful context, sure.

Comment (gryf) [reply modules/nw-kuryr-cleanup.adoc:61]: Added comment on this in the note section.

**Comment by kalexand-rh** [comment]
@gryf, is this PR ready for QE review?

**Comment by jmanthei** [comment]
@eurijon please QE review

**Comment by gryf** [comment]
@kalexand-rh yes, it is.

File: modules/nw-kuryr-migration.adoc [hunk 3]
```diff
@@ -250,24 +250,6 @@ You cannot change the service network address block during the migration.
 You cannot use any CIDR block that overlaps with the `100.64.0.0/16` CIDR block because the OVN-Kubernetes network provider uses this block internally.
 ====
 
-. Verify that the Multus daemon set rollout is complete by entering the following command:
```
Comment (imatza-rh) [inline modules/nw-kuryr-migration.adoc:253]: Hey @gryf , the Multus verification still exists in the OpenShift-SDN to OVN-Kubernetes migration procedure. [0]
How does this kind of migration different? Should the Multus verification should be deleted from there as well? 

[0] https://docs.openshift.com/container-platform/4.13/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#nw-ovn-kubernetes-migration_migrate-from-openshift-sdn

File: modules/nw-kuryr-migration.adoc [hunk 4]
```diff
@@ -286,7 +268,7 @@ done
 If SSH access is not available, you can use the `openstack` command:
 [source,terminal]
 ----
-$ for name in $(openstack server list --name ${CLUSTERID}\* -f value -c Name); do openstack server reboot $name; done
+$ for name in $(openstack server list --name "${CLUSTERID}*" -f value -c Name); do openstack server reboot "${name}" done
```
Comment (imatza-rh) [inline modules/nw-kuryr-migration.adoc:271]:
```suggestion
$ for name in $(openstack server list --name "${CLUSTERID}*" -f value -c Name); do openstack server reboot "${name}"; done
```

File: modules/nw-kuryr-migration.adoc (hunk 3 above)
Comment (gryf) [reply modules/nw-kuryr-migration.adoc:253]: Yes, it's there, but it'll be changed AFAIK.

File: modules/nw-kuryr-migration.adoc (hunk 4 above)
Comment (gryf) [reply modules/nw-kuryr-migration.adoc:271]: Nice catch, thanks!

File: modules/nw-kuryr-cleanup.adoc [hunk 5]
```diff
@@ -12,9 +12,15 @@ plugin, you must clean up the resources that Kuryr created previously.
 [NOTE]
 ====
 The clean up process relies on a Python virtual environment to ensure that the package versions that you use support tags for Octavia objects. You do not need a virtual environment if you are certain that your environment uses at minimum:
+
 * `openstacksdk` version 0.54.0
```
Comment (skrthomas) [inline modules/nw-kuryr-cleanup.adoc:16]:
```suggestion
* The `openstacksdk`Python package version 0.54.0
```

Comment (skrthomas) [reply modules/nw-kuryr-cleanup.adoc:16]: Would it be acurate to say something like the above? I think this needs a "the" and I saw this elsewhere: https://docs.openshift.com/container-platform/4.13/networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.html#nw-kuryr-migration_migrate-from-kuryr-sdn

File: modules/nw-kuryr-cleanup.adoc [hunk 6]
```diff
@@ -12,9 +12,15 @@ plugin, you must clean up the resources that Kuryr created previously.
...
+
 * `python-openstackclient` version 5.5.0
-* `python-octaviaclient` version 2.3.0 
+
+* `python-octaviaclient` version 2.3.0
```
Comment (skrthomas) [inline modules/nw-kuryr-cleanup.adoc:20]: Same comment here

File: modules/nw-kuryr-cleanup.adoc [hunk 7]
```diff
@@ -12,9 +12,15 @@ plugin, you must clean up the resources that Kuryr created previously.
...
+
+If you decide to use those particular versions, be sure to pull `python-neutronclient` prior to version 9.0.0, as it will prevent you from accessing trunks.
```
Comment (skrthomas) [inline modules/nw-kuryr-cleanup.adoc:22]:
```suggestion
If you decide to use these versions, be sure to pull `python-neutronclient` prior to version 9.0.0, as it prevents you from accessing trunks.
```

Comment (skrthomas) [reply modules/nw-kuryr-cleanup.adoc:22]: Avoid using future tense

Comment (skrthomas) [reply modules/nw-kuryr-cleanup.adoc:22]: Can "those particular" be swapped for "these" 🤔 I think that makes it clearer that you're referring to the preceding list.

**Review by skrthomas — COMMENTED** [review]
A few comments for you! Nice work :)

File: modules/nw-kuryr-cleanup.adoc (hunk 7 above)
Comment (gryf) [reply modules/nw-kuryr-cleanup.adoc:22]: Ack

**Review by imatza-rh — COMMENTED** [review]
/lgtm

**Comment by jmanthei** [comment]
/label peer-review-needed

Please validate previous peer review :) LGTM

**Review by lpettyjo — COMMENTED** [review]
You need to squash your two commits to one. See OCP Docs Manual for procedure https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#heading=h.w1w2pk9teagi. Otherwise, LGTM.

**Review by maxwelldb — APPROVED** [review]
Looks fine AFAICT. 

@gryf Not impacted by this PR, but here: https://github.com/openshift/openshift-docs/pull/62843/files#diff-a5f303c71c47cb61d604b9d1582a44d1211beea8da2c0ca9d49c8f9c5ba4a63fR173

Can we alter this step? The sentences there seem contradictory. 

image

**Comment by gryf** [comment]
Sure, we can change that.

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**